pkg/auth/dependency/forgotpwdemail/template.go

```diff
@@ -29,6 +29,17 @@ Thanks.`,
 var TemplatePasswordEmailHTML = template.Spec{
 	Type:   TemplateItemTypeForgotPasswordEmailHTML,
 	IsHTML: true,
+	Default: `<!DOCTYPE html>
+<html>
+<body>
+<p>Dear {{ .email }},</p>
```

Comment:
I think this is actually a pretty good example / idea of why if we put the first login ID as display name in metadata could be quite useful.

e.g. I would change it to:

<p> Dear {{ .metadata.displayID }}, </p> 

(assume `.metadata.displayID` is the way we put a metadata var here...) It would works really well...

Reply:
The related feature issue is here https://github.com/SkygearIO/features/issues/388

I guess we don't have anything to do at this stage.